### ISSUE TITLE
Bug: Toolbar preview keyboard shortcuts not working

### ISSUE THREAD

**Issue by alphacoma18** [uiwjs/react-md-editor#441]
Previously, I added keyboard shortcuts to all the toolbar tool titles. It works fine for the format tools but does not work for the 4 view modes (Live, Fullscreen, etc...). I've noticed that it prevents default correctly but does not execute the command. @jaywcjlove

**Comment by jaywcjlove** [comment]
@alphacoma18 Upgrade `v3.19.1`

**Comment by alphacoma18** [comment]
@jaywcjlove Good day. I see the update and it's very convenient and awesome! However, there is a bug within that if I switch from other view modes `(ctrl + 7, 8, and 10)` to preview mode `(ctrl + 9)`, we cannot switch to other view modes using keyboard shortcuts anymore. I noticed that this is because the textarea editor is where the shortcuts/event listeners are binded to but it is missing from the DOM (probably had `display: none`)

I do not know yet how to fix this but I think modifying the div to have onKeyUp (via content editable) or/and onFocus event listener will do the trick

On another note, on the browser developer tools DOM element div `class="w-md-editor-content"`, when I added `contenteditable="true"`, it allowed me to directly edit the contents inside while keeping the formatting. **Possibly a great game-changing feature** if implemented.
